OutOfSchool/OutOfSchool.AuthServer.Tests/Services/CommonMinistryAdminServiceTests.cs

```diff
@@ -122,4 +180,34 @@ private async Task Seed()
         context.Add(new User { Id = string.Empty, FirstName = string.Empty, LastName = string.Empty, Email = string.Empty });
         await context.SaveChangesAsync();
     }
+
+    private (User, AreaAdminBaseDto, AreaAdmin) CreateTestData()
+    {
+        var user = new User
+        {
+            FirstName = "Іван",
+            LastName = "Петренко",
+        };
+
+        var areaAdminBaseDto = new AreaAdminBaseDto
+        {
+            UserId = string.Empty,
+            FirstName = "Іван",
+            LastName = "Петренко",
+            Email = "test@example.com",
+            PhoneNumber = "985646549",
+            CreatingTime = DateTimeOffset.Now,
+            InstitutionId = Guid.NewGuid(),
+            CATOTTGId = 28675,
+        };
+
+        var areaAdmin = new AreaAdmin
+        {
+            InstitutionId = areaAdminBaseDto.InstitutionId,
+            User = user,
+            UserId = string.Empty,
+        };
```

Comment:
(Test design) I believe we have related test data generators. Please consider using them (and create one if any is missing).

Reply:
Fixed. Thank you!